qiskit/synthesis/clifford/clifford_decompose_layers.py

```diff
@@ -267,7 +269,7 @@ def _decompose_graph_state(cliff, validate, cz_synth_func):
     """
 
     num_qubits = cliff.num_qubits
-    rank = _compute_rank(cliff.stab_x)
+    rank = compute_rank((cliff.stab_x).astype(bool))
```

Comment:
Same comment as above
```suggestion
    rank = compute_rank((cliff.stab_x).astype(bool, copy=False))
```